spec/Section 5 -- Validation.md

```diff
@@ -248,6 +252,20 @@ subscription sub {
 }
 ```
 
+```graphql counter-example
```

Comment:
Is it worth documenting the expected behavior of sending a single document with multiple subscription operations and an operation name?

Reply:
Sure - I can add that